README.md

```diff
@@ -1,5 +1,10 @@
 # Manageiq::Loggers
```

Comment:
Do you want to also update this to `ManageIQ::Loggers`?

Reply:
I was going to let you update the README because it needs a lot of stuff.